xbmc/Application.cpp

```diff
@@ -3210,7 +3210,8 @@ PlayBackRet CApplication::PlayStack(const CFileItem& item, bool bRestart)
       }
     }
 
-    *m_itemCurrentFile = item;
+    CFileItemPtr tmp(new CFileItem(item));
+    m_itemCurrentFile = tmp;
```

Comment:
@xhaggi I don't think that is the same because now you're not creating a new object, just sharing the pointer.